airflow/bin/airflow

```diff
@@ -1,9 +1,30 @@
 #!/usr/bin/env python
 import logging
 import os
+import socket
+import requests
 from airflow import configuration
 from airflow.bin.cli import CLIFactory
 
+def get_private_ip(name=''):
+    r = requests.get("http://169.254.169.254/latest/meta-data/local-ipv4")
```

Comment:
We could, but I see no reason to make it configurable unless it could change. This is the static address from which EC2 instances vend metadata - see the [AWS docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) for details.